src/Compilers/VisualBasic/Portable/CodeGen/EmitExpression.vb

```diff
@@ -1022,7 +1022,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
             EmitSymbolToken(method, [call].Syntax)
             If Not method.IsSub Then
                 EmitPopIfUnused(used)
-            ElseIf _optimizations = OptimizationLevel.Debug Then
+            ElseIf _ilEmitStyle = OptimizationLevel.Debug Then
```

Comment:
I have fixed all the cases, but not having an error on _ilEmitStyle! seem sto be a bug. Will file that separately with a smaller repro.